java/src/com/liferay/nativity/control/win/WindowsNativityUtil.java

```diff
@@ -65,15 +65,17 @@ private static void _load() {
 
 			_loaded = true;
 
+		    System.out.println("Successfully loaded " + nativityDllName);
 			_logger.trace("Loaded WindowsNativityUtil DLL");
```

Comment:
Also, if the DLL is now named BoxDesktopWindowsUtil we should change all of the logs.  ```grep``` through the entire proj to make sure theres none left behind